dashboard/test/ui/support/browser_helpers.rb

```diff
@@ -82,6 +82,12 @@ def check_window_for_js_errors(check_reason_description)
         puts "DEBUG: [#{check_reason_description}] No JS errors found on current page."
       end
     end
+  rescue
+    # We're not currently failing any tests based on JS errors showing up, so
+    # this is just a debugging tool.
+    # We're getting intermittent timing errors that have to do with SauceLabs
+    # going away before we can check for JS errors.
+    # We don't want that to cause test runs to fail, so ignore exceptions for now.
```

Comment:
Seems like it would make sense to print a message indicating that this has happened, even just `puts 'DEBUG: Error capturing JS errors'`.